plugin-server/src/main/ingestion-queues/session-recording/services/console-logs-ingester.ts

```diff
@@ -30,9 +30,10 @@ function deduplicateConsoleLogEvents(consoleLogEntries: ConsoleLogEntry[]): Cons
     const deduped: ConsoleLogEntry[] = []
 
     for (const cle of consoleLogEntries) {
-        if (!seen.has(cle.message)) {
+        const fingerPrint = `${cle.log_level}-${cle.message}`
```

Comment:
the only real change since last review... 

Deduplication didn't work at all 🙈 🤦